### DISCUSSION TITLE
Existence of peaks to the right of the molecular ion (M+) peak

### DISCUSSION THREAD

**Discussion by 4psireal2** [qcxms/QCxMS#63] [Q&A] [answered]
I tested the EI mode for Xanax (C17H13ClN4) which has a molar mass of 308.77 g·mol−1. The resulting MS (obtained with PlotMS) does show a molecular peak at an m/z ratio of  308.08. However, there exist peaks for m/z ratio up to 313.09 with some having significant intensity (for ex: [(309.09, 18.7), (310.08, 32.4), (311.08, 6.04)] where [(m/z_ratio, intensity)]).  The only explanation that comes to mind is that this is due to the isotopic abundance of the molecular atoms. However, the high intensity casts doubt on it and I wonder whether I should consider this effect as a simulation artefact and filter these peaks out.

**Comment by JayTheDog** [comment, marked as answer]
The M+ peak is always the one you put in as a structure, there can be no artificial larger peak due to the QCxMS calculation. So it definitely stems from the isotope pattern. You can switch off plotting the isotope pattern by using the `-p` option to check if this is really the case. The high intensity comes from the large intensity of surviving molecular ions, check if the fragment ions might be too low in intensity too. If so, either increase the impact energy or the sampling time.

**Reply by 4psireal2** [reply]
By sampling time, do you mean the parameter `tmax` ?

**Reply by JayTheDog** [reply]
Exactly. However, while running a simulation longer enables a better distribution of the internal/excess energy, it does not guarantee that the provided energy is large enough for fragmentation. If you want to increase the electron beam impact energy, use `eimp0` and set it to e.g. 80. Another factor would be increasing the `ieeatm` to more than 0.6, as this influences the energy distribution per bond. The values (70eV and 0.6 eV/atm) are parameters set to provide the best results for *most* compounds, but it does not necessarily mean that it will be the correct settings for your compound as well.

**Reply by 4psireal2** [reply]
Thank you for the advice ! Now I'd like to have better guesses for `eimp0` and `ieeatm` than the default settings. Which simulation (I'm familiar with the simulation of thermal fragmentation using xtb MTD simulations ) should I run to get these guesses?

**Comment by tobigithub** [comment]
@psireal9 can you post the final spectrum? What is the cosine similarity score when compared to the experimental spectrum here: https://webbook.nist.gov/cgi/cbook.cgi?ID=C28981977&Mask=200 
The JCAMP spectrum (ASCII) is available here: Download [spectrum](https://webbook.nist.gov/cgi/cbook.cgi?JCAMP=C28981977&Index=0&Type=Mass) in JCAMP-DX format.

![image](https://user-images.githubusercontent.com/2569565/234751969-e8fbbd73-0125-47e4-ba62-3ced83b85f23.png)
![image](https://user-images.githubusercontent.com/2569565/234752026-af885367-a2e2-4ba7-9fc3-57904dac4949.png)

**Reply by 4psireal2** [reply]
here is what i got for the following setting:  `xtb, ntraj 300, tmax 10, tinit 500, ieeatm 0.6` without the calculation of isotope pattern

15.023475          1.412298591549295601
   25.007825          0.062382253521126765
   26.015650          0.634727042253521057
   27.023475          0.323337323943661992
   28.018724          0.012717323943661974
   39.010899          0.985619014084506917
   40.018724          5.003713098591548558
   40.031300          1.215785211267605526
   41.026549          1.217022112676056311
   42.034374          2.459041126760562967
   46.968852          0.001483661971830986
   50.015650          1.162013521126760596
   51.023475          2.620191126760563538
   52.018724          1.033529436619718300
   53.013973          0.282815211267605671
   53.039125          3.241170704225352583
   54.021798          0.985286056338028127
   55.029623          0.053753239436619721
   56.037448          0.007997887323943660
   59.976677          0.013358732394366198
   60.984502          1.406353380281690102
   62.015650          0.308824084507042240
   63.023475          1.438605915492957799
   64.018724          1.634153943661971864
   64.031300          1.139745352112676136
   65.026549          0.703218873239436659
   65.039125          1.407540563380281773
   66.034374          3.238982535211267777
   66.046950          0.006390845070422535
   67.029623          0.065163521126760562
   67.042199          0.112331126760563366
   68.037448          0.749077887323943648
   71.976677          0.027727183098591551
   72.984502          1.359449859154929552
   74.015650          2.714661830985915625
   75.023475          1.185642394366197117
   76.018724          1.260572394366197280
   76.031300          3.307410704225352660
   77.026549          1.228638028169014129
   77.039125         15.829503521126762067
   79.029623          1.461318591549295665
   79.042199          1.807403239436619735
   80.024872          0.004212957746478873
   81.032697          0.003682253521126761
   82.040522          0.003800563380281691
   83.976677          0.002477183098591549
   84.984502          1.876314507042253776
   85.992327          0.412677323943662022
   86.015650          1.405108732394366289
   87.023475          1.172564929577464721
   89.013973          0.003135352112676056
   89.039125          1.531458028169014218
   90.034374          2.374505633802816984
   91.042199          3.639697746478873341
   92.037448          0.510392816901408475
   95.048347          0.862509295774648077
   97.979751          0.201901408450704228
   98.015650          0.238320985915492944
   99.010899          1.296667746478873395
   99.023475          0.261037183098591585
  101.026549          1.211086901408450878
  102.021798          0.110000704225352131
  102.034374          2.500007323943662207
  103.042199          4.345335211267604691
  103.054775          1.250981126760563278
  104.050024          2.952605352112676051
  105.045273          0.736571267605633850
  108.984502          0.041226478873239436
  109.051421          0.098099436619718330
  109.992327          5.715788873239437073
  110.987576          0.049997887323943668
  111.000152          0.456435492957746491
  113.039125          1.407810281690140775
  114.021798          1.320423802816901304
  115.029623          0.142554225352112679
  115.042199          1.164504084507042236
  116.050024          5.419429014084507834
  117.057849          6.882594788732395052
  118.065674          1.408408028169014115
  120.056172          0.521772676056338036
  121.051421          0.004586197183098591
  123.995401          2.579694366197183175
  125.003226          0.072704507042253522
  125.026549          0.003530000000000000
  126.034374          1.407990563380281612
  127.042199          1.376583098591549259
  128.050024          1.340114507042253544
  129.057849          0.951626197183098488
  130.053098          0.046343943661971830
  134.987576          0.716559295774647831
  135.000152          1.234362112676056444
  135.995401          1.585139014084507192
  136.007977          1.407930563380281663
  137.003226          1.304658028169014106
  137.015802          1.408249718309859233
  138.046950          1.408402253521126690
  140.050024          0.920981690140845077
  142.053098          2.868220422535211256
  142.065674          1.163629577464788900
  143.060923          1.408040845070422487
  149.003226          1.335657464788732263
  149.998475          1.356646197183098534
  150.011051          1.389062816901408537
  151.006300          0.158240000000000020
  151.042199          1.721706056338028201
  151.054775          1.408165774647887281
  154.040522          0.016723661971830988
  155.048347          1.314972253521126788
  155.060923          0.936193380281690302
  161.003226          0.027366197183098594
  161.998475          0.049140000000000003
  162.046950          1.404222253521126840
  163.006300          1.110799154929577481
  163.054775          4.203453239436619526
  164.014125          0.712956901408450694
  165.021950          1.407309295774647806
  167.073499          1.408450563380281739
  170.071822          1.408352394366197302
  171.000152          1.408303239436619725
  174.023627          1.408438028169014089
  175.018876          1.408449859154929484
  176.014125          0.301872535211267612
  176.062600          1.408434225352112712
  177.021950          1.126361549295774589
  177.034526          4.200618028169013307
  177.057849          1.403608028169014199
  178.040522          1.408421549295774566
  178.065674          2.816630140845070240
  180.068748          1.408449718309859211
  187.006300          2.243117746478873187
  190.017199          0.007303098591549296
  191.025024          1.396053943661971886
  192.068748          1.406401690140845151
  193.063997          0.029285492957746479
  196.074896          1.365628309859154843
  197.082721          1.154475774647887309
  199.031452          2.736711830985915306
  200.026701          0.437055211267605659
  200.039277          1.333353380281690148
  203.037600          1.399538309859154950
  204.020273          0.066227746478873237
  204.032849          0.087903802816901411
  204.068748          2.816254084507042066
  205.028098          0.157469577464788735
  205.040674          1.442129014084506888
  211.018876          1.408394084507042177
  212.026701          1.348077887323943624
  213.021950          1.399979718309859233
  213.034526          5.485925070422535299
  219.043748          1.334333943661971889
  225.034526          1.408450704225352235
  226.029775          0.282855211267605655
  226.042351          2.816663239436620003
  227.037600          4.274410563380282468
  227.050176          2.813083802816901624
  231.043748          2.911042112676056171
  232.087472          2.816901408450704469
  239.037600          1.408450422535211244
  239.050176          2.816901408450704469
  240.032849          2.814736619718309907
  240.045425          1.408450704225352235
  241.053250          1.408332535211267666
  248.106196          1.408450563380281739
  250.029775          1.408444647887324042
  253.053250          8.413439014084508116
  254.061075          1.408450704225352235
  257.059398          1.408184225352112628
  266.048499          1.408450704225352235
  267.056324         23.943613098591548294
  268.064149          2.816791830985915457
  273.114021          8.450704084507041358
  280.076725          5.633802816901408939
  282.067223          1.408449436619718220
  291.043748          1.408419859154929510
  293.059398          1.405892394366197173
  305.059398          1.408450563380281739
  306.067223          1.408450704225352235
  307.075048         21.126760422535213024
  308.082873        100.000000000000000000

**Reply by JayTheDog** [reply]
As you see, the isotope pattern vanishes. The intensity of the M+ peak can be reduced as I described before. Otherwise, this looks good, but as you see, you only get an averaged value without isotope pattern (because I didn't implement this feature in great detail)

**Comment by tobigithub** [comment]
Also isotopic pattern are here
````
Isotopic Abundances for C17H13ClN4
  Mass/Charge	Fraction 	Intensity
   308.08287	0.6206963	 100.00
   309.08287	0.1245067	  20.06
   310.08287	0.2102348	  33.87
   311.08287	0.0405059	   6.53
   312.08287	0.0038191	   0.62
   313.08287	0.0002292	   0.04
   314.08287	0.0000097	   0.00
````
![image](https://user-images.githubusercontent.com/2569565/234752451-28b7e474-0fc3-4f24-a4df-1640dc0e1064.png)
